### ISSUE TITLE
Colour selections fail accessibility guidelines

### ISSUE THREAD

**Issue by papadeltasierra** [PowerShell/PowerShell#20457]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://learn.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

PowerShell 7.3.7.  Use some commands and trigger "hint" text.  For example see the image below:

![image](https://github.com/PowerShell/PowerShell/assets/18214079/83896b70-679d-4fc4-9ddc-bbf9ae0a1a3e)

As someone who is slightly colourblind, I can barely see the help text and it certainly does not pass accessibility guidelines according to Accessibility Insights for Windows.

I have tried changing the colour scheme but they all seem to suffer from similar issues and I could not find one that made all the test have a good contrast.

### Expected behavior

```console
Text should all be easily readable, albeit perhaps with a need to select a "high contrast" colour scheme.
```

### Actual behavior

```console
Various different text colours, of which the help text is one example, are hard if not impossible to read.
```

### Error details

```console
None.
```

### Environment data

```powershell
PS > $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.3.7
PSEdition                      Core
GitCommitId                    7.3.7
OS                             Microsoft Windows 10.0.22621
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

PS >
```

### Visuals

![image](https://github.com/PowerShell/PowerShell/assets/18214079/0e22dad4-106d-4094-82fe-309f47680ac0)

**Comment by 237dmitry** [comment]
> I have tried changing the colour scheme

I think you can change PSReadline options:

```
Get-PSReadlineOption
```

```
Set-PSReadlineOption -Color @{

  ListPredictionColor = "Green"       
  ListPredictionSelectedColor = "Red"
  ListPredictionTooltipColor = "`e[38;5;95m"
}
```

**Comment by papadeltasierra** [comment]
@237dmitry Thank you so much!  Excellent tip!  Doesn't let MS off the hook for not having a default accessibility compatible scheme of course but does help work around it ;-).
